xml/System.CommandLine/IdentifierSymbol.xml

```diff
@@ -0,0 +1,169 @@
+            </summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="protected IdentifierSymbol (string name, string? description = default);" />
+      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(string name, string description) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.CommandLine.IdentifierSymbol.#ctor(System.String,System.String)" />
+      <MemberSignature Language="VB.NET" Value="Protected Sub New (name As String, Optional description As String = Nothing)" />
+      <MemberSignature Language="F#" Value="new System.CommandLine.IdentifierSymbol : string * string -&gt; System.CommandLine.IdentifierSymbol" Usage="new System.CommandLine.IdentifierSymbol (name, description)" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="name" Type="System.String" />
+        <Parameter Name="description" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="name">The name of the symbol.</param>
+        <param name="description">The description of the symbol, which is displayed in command line help.</param>
+        <summary>
+            Initializes a new instance of the <see cref="T:System.CommandLine.IdentifierSymbol" /> class.
+            </summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="AddAlias">
+      <MemberSignature Language="C#" Value="public void AddAlias (string alias);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig instance void AddAlias(string alias) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.CommandLine.IdentifierSymbol.AddAlias(System.String)" />
+      <MemberSignature Language="VB.NET" Value="Public Sub AddAlias (alias As String)" />
+      <MemberSignature Language="F#" Value="member this.AddAlias : string -&gt; unit" Usage="identifierSymbol.AddAlias alias" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; void AddAlias(System::String ^ alias);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="alias" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="alias">The alias to add.</param>
+        <summary>
+            Adds an alias. Multiple aliases can be added, most often used to provide a shorthand alternative.
```

Comment:
```suggestion
            Adds an alias to provide a shorthand alternative.
```

Reply:
There are other uses for aliases, such as to provide an alternate spelling of a long-form option name (--serialize, --serialise, for example), or to simulate case-insensitivity.

Reply:
Thanks, I updated the suggestion.